_posts/2025-07-21-vox-pupuli-has-grown-up.md

```diff
@@ -0,0 +1,75 @@
+I gave an ignite talk about Vox Pupuli, and the slides are [still around](https://bastelfreak.de/cfgcamp/#1).
+
+People that had to endure my more recent talks might notice that I still use the same slide styleset as almost a decade ago, which I got from Mina back then.
+This is of course a homage to Mina, I'm not just lazy.
+
+My ignite talk went well and I was happy to meet so so many people back then. It was a very friendly event with a welcoming community.
+
+nibalizer did the second keynote in at Config Manangement Camp Berlin.
+He went on stage with a neck tie.
+
+<img alt="the awesome, the one any only, nibalizer!" src="{{ site.url }}{{ site.baseurl }}/static/images/IMG_9128.JPG" width="50%" height="50%" />
+
+The message back then: Open source is serious business.
+And in a serious business you sometimes wear ties.
+
+Almost a decade ago he already realized how important open source communities are and how much value they produce as a collective, and how much every individual contributor benefits from it.
+
+### Fast forward, July 2025
+
+In the past months, I had the pleasure to organize the first Vox Pupuli conference, [VoxConf](https://voxpupuli.org/voxconf/).
+This was the first conference for many community members.
+The goal was to give them the same welcoming feeling as I had on my first conference.
+
+The event was hosted by Netways, the same company that organized my puppet training back then.
+The recommended VoxConf hotel was the Park Inn hotel, where the training took place.
+My trainer back then was Lennart Betz, and I'm more than happy that we could convince him to not only give a talk at VoxConf, but also provide an awesome city tour the day before.
+The picture above was taken by Dirk Götz, who works for Netways and is deeply involved in the Foreman ecosystem.
+Vox Pupuli and Foreman are closely collaborating since years, so it made sense to me to host our event alongside the Foreman birthday party.
+ffrank wasn't much active in the community for some time, so I was very happy to see him again on VoxConf.
+And of course I reused again the slide styleset from Mina.
+
+And of course this had to happen:
+
+<img alt="bastelfreak with a neck tie" src="{{ site.url }}{{ site.baseurl }}/static/images/pxl_20250717_065650108.jpg" width="50%" height="50%" />
+
+Vox Pupuli has grown up.
+We are an active community that still grows after all those years.
+We maintain the majority of the puppet testing ecosystem, the most used modules and with OpenVoxProject we even have a "side project" which now provides open source packages a successors for the [deprecated puppet agent/server/db](https://www.puppet.com/blog/open-source-puppet-updates-2025).
+
+I'm very grateful for my personal journey in the past decade and I feel honoured that you allow me to serve in the PMC since it's establishment.
```

Comment:
```suggestion
I'm very grateful for my personal journey in the past decade and I feel honoured that you allow me to serve in the PMC since its establishment.
```